catalogs/usfimr/build_catalog.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if test -f /tmp/USFIMR_all.zip; then
+  echo "/tmp/USFIMR_all.zip already downloaded. Continuing..."
+else
+  echo "/tmp/USFIMR_all.zip not downloaded. Downloading..."
+  wget ftp://guest:guest@sdml-server.ua.edu/USFIMR/USFIMR_all.zip -O /tmp/USFIMR_all.zip
+fi
+
+if test -d /tmp/USFIMR_all; then
+  echo "/tmp/USFIMR_all.zip already unzipped. Continuing..."
+else
+  echo "Unzipping /tmp/USFIMR_all.zip..."
+  unzip /tmp/USFIMR_all.zip -d /tmp
+fi
+
+if test -f /tmp/latlng/USFIMR.shp; then
+  echo "USFIMR already reprojected. Continuing..."
+else
+  echo "Reprojecting USFIMR..."
+  mkdir -p /tmp/latlng
+  # wildcard used here to accomodate dated filenames
+  ogr2ogr /tmp/latlng/USFIMR.shp -t_srs "EPSG:4326" /tmp/USFIMR_all/USFIMR_*.shp
```

Comment:
@CloudNiner this is the suspect line

Reply:
When I load up the `/tmp/USFIMR_all/USFIMR_....shp` I see this:

<img width="800" alt="Screen Shot 2020-08-14 at 9 39 05 AM" src="https://user-images.githubusercontent.com/1158084/90255236-0271a000-de12-11ea-8d9c-6bb28a874e2d.png">